scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/conflicts/MissingActivityTemplateConflict.java

```diff
@@ -75,6 +80,11 @@ public Optional<Duration> getTotalDuration(){
     return totalDuration;
   }
 
+  @Override
+  public ScheduleAt scheduleAtEarliest() {
```

Comment:
I think these methods should just be called `scheduleAt()` because it is returning an enum not a boolean.